tests/Support/SupportHtmlStringTest.php

```diff
@@ -7,17 +7,29 @@
 
 class SupportHtmlStringTest extends TestCase
 {
+    /**
+     * @param  string  $string
+     * @return \Illuminate\Support\HtmlString
+     */
+    protected function htmlString($string = '')
```

Comment:
Please remove this method and just write `(new HtmlString(...))->`.

Reply:
Wrote the same implementation as in:
https://github.com/laravel/framework/blob/1f163d471b973b237772bb11cdcb994aadd3d530/tests/Support/SupportStringableTest.php#L8-L17

Reply:
Doesn't matter. It's not needed here.

Reply:
Please don't make necessarily changes to the code.